app.py

```diff
@@ -1,143 +1,58 @@
-from time import sleep, time
-from gpiozero import Button, DigitalOutputDevice, PWMOutputDevice
+# -*- coding: utf-8 -*-
 from signal import pause
-from threading import Timer
+from time import sleep
 
+from record import record_to_file, play_from_file
+from speak import speak
+from beep import beep
+from phone import Dial, Flash_button, Receiver
 
-# settings
-DIAL_TIMEOUT = 1.5
 
-# config
-DIAL_CYCLE = 8
-DIAL_SIGNAL = 7
-HOOK = 13
-BUTTON_LOW = 19
-BUTTON_HIGH = 26
-BELL_1A = 27
-BELL_2A = 22
-BLINK = 21
+RECEIVER_GPIO = 13
+FLASH_BUTTON_LOW_GPIO = 19
+FLASH_BUTTON_HIGH_GPIO = 26
+DIAL_CYCLE_GPIO = 8
+DIAL_SIGNAL_GPIO = 7
 
-# TODO: wrap whole dial stuff in own class(?) to start/stop and set callback
-dial_cycle = Button(DIAL_CYCLE)
-dial_signal = Button(DIAL_SIGNAL)
+def main():
+  receiver = Receiver(RECEIVER_GPIO)
+  receiver.when_picked_up(record_some_words)
```

Comment:
Does it make sense to use [events](https://pypi.python.org/pypi/Events) in my classes instead? Would be cool to be able to add multiple callbacks and remove them afterwards. Or what is the common way to do something like that?